packages/core/streams/ai-stream.ts

```diff
@@ -13,6 +27,42 @@ export interface AIStreamCallbacks {
   onStart?: () => Promise<void>
   onCompletion?: (completion: string) => Promise<void>
   onToken?: (token: string) => Promise<void>
+
+  /**
+   * Only applicable for OpenAI
```

Comment:
Can we move this type to the `OpenAIStream()` constructor instead? So it becomes `AIStreamCallbacks & { experimental_onFunctionCall?: ... }`